Definitions/Chord

```diff
@@ -0,0 +1,29 @@
+class Chord:
+    def __init__(self, symbol, degrees, next, inversion):
+        self.symbol = symbol
+        self.degrees = degrees
+        self.next = next
+        self.inversion = inversion
+        
+    def hasMelodicNote(self, note):
+        result = note in self.degrees
+        return result
+    
+    def getSuccessors(self):
+        return self.next
+    
+    def hasParallels(self, nextChord, note1, note2):
```

Comment:
Let's revisit this when we have a more solid footing on what our data will look like. I think there is a lot to consider here (as your comment also suggests), and continuously revising this as our understand of the system grows seems like a waste of effort when we could just revise it more as we get further along.